.github/workflows/ci.yml

```diff
@@ -14,17 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: pnpm/action-setup@v2.0.1
         with:
-          node-version: '14.x'
-      - name: Cache ~/.pnpm-store
-        uses: actions/cache@main
+          version: 6
```

Comment:
Maybe instead of `pnpm/action-setup`, we can directly run `npm i -g pnpm`. vite-plugin-svelte also [does this](https://github.com/sveltejs/vite-plugin-svelte/blob/c086c9f7db99b87a53dcb04e8d07d8ee7846798b/.github/workflows/ci.yml#L30)

Reply:
At the moment I don't think there are any _significant_ benefits, as far as I know the only special thing is does is during the `post`-phase where it runs `pnpm store prune`. However it opens the door for easier integration in the future if they add caching support directly to the crate for example, which may give benefits more specialized for `pnpm` than what `actions/setup-node` can do.

Reply:
I've personally switched to installing pnpm with npm. The pnpm action still produces intermittent installation failures on windows and I've found that installing with npm is typically a little faster (the installation is probably the same or faster with the action but it needs to checkout a repo in order to run it). The CI code is also shorter, a small benefit but nice nonetheless.

It probably doesn't make much difference either way